src/consts.ts

```diff
@@ -1 +1,6 @@
+import { PublicKey } from "@solana/web3.js";
+
 export const MAX_SIGNATURES_PER_CALL = 300;
+export const PROMOTED_POOL = new PublicKey(
```

Comment:
It will start with only one pool being promoted/whitelisted, but in the future it can grow. You can store whitelisted pools in an array for easier future adoption.